cvpysdk/commcell.py

```diff
@@ -164,6 +165,7 @@ def __init__(self, webconsole_hostname, commcell_username, commcell_password=Non
             UserGroups,
             WorkFlow,
             ClientGroups,
+            Datacube

```

Comment:
oops, merge conflict. Will fix. I wonder how it did not throw an error when I tested